delta/plugins/storage/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/storage/files/model/Digest.scala

```diff
@@ -21,19 +21,23 @@ object Digest {
     * @param value
     *   the actual value of the digest of the file
     */
-  final case class ComputedDigest(algorithm: DigestAlgorithm, value: String) extends Digest {
-    override val computed: Boolean = true
-  }
+  final case class ComputedDigest(algorithm: DigestAlgorithm, value: String) extends Digest
 
   /**
     * A digest that does not yield a value because it is still being computed
     */
-  final case object NotComputedDigest extends Digest {
-    override val computed: Boolean = false
-  }
+  final case object NoDigest extends Digest
```

Comment:
This is slightly different from the other one where we were expecting it to be computed in the future